src/app/_types/sortTypes.ts

```diff
@@ -23,7 +23,7 @@ export type SortOption = {
 }
 
 export type SortableByDate = {
-  publishedOn?: DynamicBaseData['publishedOn']
+  publishedOn?: DynamicBaseData['published-on']
```

Comment:
How come we are changing this to kebab case here?

```suggestion
  publishedOn?: DynamicBaseData['published-on']
```